### PR TITLE
Referral closure at Addo

### PR DESCRIPTION
**Pull request by Andati**

### CODE REVIEW

File: app/src/main/java/org/smartregister/addo/sync/helper/AddoTaskServiceHelper.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        if (instance == null) {
+            instance = new AddoTaskServiceHelper(CoreLibrary.getInstance().context().getTaskRepository());
+        }
+        return instance;
+    }
+
+    @Override
+    protected List<String> getLocationIds() {
+        List<String> res = new ArrayList<>();
+        res.add("bb8e7fd4-f8d8-42d9-a71c-d659f9e2c64d");
```
Comment (gosso22) [inline app/src/main/java/org/smartregister/addo/sync/helper/AddoTaskServiceHelper.java:30]: @Andati - can you check this hard coded string?

File: app/src/main/java/org/smartregister/addo/sync/helper/AddoTaskServiceHelper.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        return res;
+    }
+
+    @Override
+    protected Set<String> getPlanDefinitionIds() {
+        Set<String> res = new HashSet<>();
+        res.add("5270285b-5a3b-4647-b772-c0b3c52e2b71");
```
Comment (gosso22) [inline app/src/main/java/org/smartregister/addo/sync/helper/AddoTaskServiceHelper.java:37]: And this as well

File: app/src/main/java/org/smartregister/addo/sync/helper/AddoTaskServiceHelper.java (hunk 1 above)
Comment (Andati) [reply app/src/main/java/org/smartregister/addo/sync/helper/AddoTaskServiceHelper.java:30]: Yeah, this is a constant.

File: app/src/main/java/org/smartregister/addo/sync/helper/AddoTaskServiceHelper.java (hunk 2 above)
Comment (Andati) [reply app/src/main/java/org/smartregister/addo/sync/helper/AddoTaskServiceHelper.java:37]: Yeah, this is a constant.

**Pull request merged**